### PR TITLE
New formula shotdetect

### PR DESCRIPTION
**Pull request by jjeising**
Added shotdetect head only formula. A test is currently not included as
it requires a video file to be useful.

### CODE REVIEW

File: shotdetect.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+class Shotdetect < Formula
+  desc "Detects shots and scenes from a video"
+  homepage "https://github.com/johmathe/Shotdetect"
+
```
Comment (DomT4) [inline shotdetect.rb]: Don't need the empty line here.

File: shotdetect.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  depends_on "gd"
+  depends_on "libxml2"
+  depends_on "libxslt"
+
+  def install
+    args = std_cmake_args.concat %W[
+      -DUSE_WXWIDGETS:BOOL=OFF
+    ]
+
+    mkdir "build"
```
Comment (DomT4) [inline shotdetect.rb]: You can just do:

```ruby
mkdir "build" do
  system "cmake", "..", *args
  system "make", "shotdetect-cmd"
  bin.install "shotdetect-cmd" => "shotdetect"
end
```

Comment (jjeising) [inline shotdetect.rb]: Thanks, fixed and updated.

**Comment by bfontaine** [comment]
Sorry for the late response.
We’re deprecating this repository and won’t accept new formulae. Please ask upstream for a stable release and submit to the [core repository](https://github.com/Homebrew/homebrew-core) or [maintain the formula in a tap](https://github.com/Homebrew/brew/blob/master/share/doc/homebrew/How-to-Create-and-Maintain-a-Tap.md). Thanks!

**Pull request closed without merging**